packages/design-tokens/src/js/themes/theme-light-default/colors.ts

```diff
@@ -45,23 +45,30 @@ export const componentButtonPlainStateHover = '#f1edff';
 
 export const componentButtonPrimaryBorder = '#4e2ea6';
 export const componentButtonPrimaryContent = '#fff';
+export const componentButtonPrimaryContentSelected = '#fff';
```

Comment:
Revert these tokens

Reply:
I took the liberty of throwing away commits with these changes to close the ticket within this sprint. We should sort this out with Jirka in the near future.